build.sh

```diff
@@ -155,5 +155,12 @@ configure_user(){
     echo '%wheel ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers.d/wheel-nopasswd
 }
 
+write_repo() {
```

Comment:
the name of this function seems pretty arbitrary considering `repo` is an arbitrary term here ostree repo? yum repo?, nope. none of those. 

`write_archive_info()` maybe?

Reply:
and now I get it.. **git repo**!

Still, I think the case stands for being more specific